ginjwt/jwt.go

```diff
@@ -107,6 +107,19 @@ func (m *Middleware) AuthRequired(scopes []string) gin.HandlerFunc {
 			return
 		}
 
+		// Some OIDC providers tack on a trailing slash to the iss: field even if it isn't
+		// specified. This ensures that the Issuers that are being compared are at least
+		// equivalent.
+		if ok := strings.HasSuffix(cl.Issuer, "/"); ok {
+			if chk := strings.HasSuffix(m.config.Issuer, "/"); !chk {
+				m.config.Issuer += "/"
```

Comment:
changing the Issuer in the middleware's config here doesn't look thread-safe... Is there another way of doing such comparison without having to change the config?

... though, now that I double check this, I'm getting some concerns about the thread-safety of this middleware overall. @nicolerenee wouldn't the `cachedJWKS` refreshing (which happens in `refreshJWKS`)  be problematic in parallel scenarios?